### PR TITLE
Feature badge

### PR DESCRIPTION
**Pull request by abhijitdotsharma**
Badge component has 4 type of badges - 
- Notification
- Mail
- Cart
- Avatar

badge-image

### CODE REVIEW

File: components/main.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="component.css">
+    <title>Install | nou-CL</title>
+</head>
+<body>
+    <!-- navigation bar -->
+    <a href="./alert/alert.html">Alert</a>
+    <!-- section -->
+        <!-- header -->
+        <!-- main -->
```
Comment (harshitadaswani) [inline components/main.html]: remove comments

File: components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+.badge{
+    width: min-content;
+    position: relative;
+}
+
+.badge-notification{
+    border-radius: 50%;
+    padding: 0.1rem 0.35rem;
+    background-color: blue;
+    color: white;
```
Comment (harshitadaswani) [inline components/badge/badge.css:15]: define those colors as variables and then use it

**Review by harshitadaswani — APPROVED** [review]
Refactor the suggestions mentioned and you are good to go

**Review by shaksham08 — APPROVED** [review]
Looks good. Just follow the changes which harshita suggested.

**Pull request merged**